htlc/src/components/withdrawHtlc.js

```diff
@@ -38,17 +61,22 @@ async function WithdrawHtlc() {
     amount = amount - params.fee;
```

Comment:
Why are we doing this?  I think that's why i am getting the error i posted above.

Reply:
While sending the amount from escrow to reciever's account, a fee needs to be deducted from the escrow.

Reply:
We don't need to explicitly deduct the fee here.

Reply:
Yes. I understand now. Making changes.